src/sqlast/value.rs

```diff
@@ -21,6 +21,12 @@ pub enum Value {
     Time(String),
     /// Timestamp literals, which include both a date and time
     Timestamp(String),
+    /// Time intervals
+    Interval {
```

Comment:
SQL allows an optional sign here.

Reply:
Indeed, but I haven't found a database that actually supports that yet. I'm going to omit for now because we don't seem to have a SQLSign type handy. I've left a note in the parsing code, though.